src/sql/workbench/browser/editor/tableDesigner/tableDesignerInput.ts

```diff
@@ -27,11 +27,12 @@ export class TableDesignerInput extends EditorInput {
 	constructor(
 		private _provider: TableDesignerProvider,
 		private _tableInfo: azdata.designers.TableInfo,
+		private _telemetryInfo: { [key: string]: string },
```

Comment:
the telemetry information is not used outside of the constructor, you can remove the private modifier and change it to `telemetryInfo`